repos/rust-lang/rustlings.toml

```diff
@@ -0,0 +1,10 @@
+org = "rust-lang"
+name = "rustlings"
+description = ":crab: Small exercises to get you used to reading and writing Rust code!"
+bots = []
+
+[access.teams]
+
+[access.individuals]
+# This repository probably needs a separate team?
+shadows-withal = "write"
```

Comment:
I wonder if we should just do this as part of this PR?

Reply:
It seems like sometimes we create teams just on a whim, but sometimes it requires an RFC? (https://rust-lang.zulipchat.com/#narrow/stream/301329-t-devtools/topic/mdbook.20team). I'm not sure what's the proper way.

Reply:
yeah, I've thought about this before, but I'm not even sure where a "team" here would fit under? language? infra? devtools?